### ISSUE TITLE
Forms with no inputs produce error V6 (alpha 15)

### ISSUE THREAD

**Issue by jaraquistain** [erikras/redux-form#1124]
In some instances I'm using `redux-form` for forms that technically have no inputs, just a submit button. The reason for this is consistency in handling server-side errors.

I'm using alpha 15 and for my forms with no inputs I'm getting the following error:

```javascript
reduxForm.js?c2ae:400 Uncaught TypeError: Cannot read property 'map' of undefined
```

specifically, this line:

```javascript
this.props.registeredFields.map(function (field) {
  return getIn(field, 'name');
});
```

Is there any harm in just having an empty array be the default value for `this.props.registeredFields`?

**Comment by erikras** [comment]
Fixed. I will comment again when the fix is published.

**Comment by erikras** [comment]
Published in [`v6.0.0-rc.1`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.1).
